docs/apm/agent-configuration.asciidoc

```diff
@@ -0,0 +1,34 @@
+[role="xpack"]
+[[agent-configuration]]
+=== APM Agent configuration
+
+beta[] APM Agent configuration allows you to fine-tune your agent configuration directly in Kibana.
+Best of all, changes are automatically propagated to your APM agents so there's no need to redeploy.
+
+To get started, simply choose the service and environment you wish to configure.
+Agents will periodically poll APM Server and apply any configuration changes.
+APM Agent configuration takes precedence over any settings set locally in each agent.
+
+[role="screenshot"]
+image::apm/images/apm-agent-configuration.png[APM Agent configuration in Kibana]
+
+IMPORTANT: As this feature is in Beta, a limited number of configuration settings are supported.
```

Comment:
might be relevant to recommend users to watch agent logs to confirm that configuration has been applied, as there is no way in the UI to know that (it might change).